build.rs

```diff
@@ -29,6 +29,7 @@ impl Version {
         vec![
             Version::new(1, 2, 1),
             Version::new(1, 5, 0),
+            Version::new(1, 5, 3),
```

Comment:
Please also add a `1.5.3` step to the CI in https://github.com/rust-pcap/pcap/blob/main/.github/workflows/01-build-and-test-unix.yml.

Reply:
Done